### ISSUE TITLE
Nan or Inf appears in the loss when training to 34 epoch, pretraining the  backbone

### ISSUE THREAD

**Issue by RongkunYang** [VinAIResearch/ISBNet#6]
Dear authors,
  I train the backbone and set the same parameters as the default, trained on a single 3090 GPU, batch size=16,  lr=4e-3, when trained to 34 epoch, there exists a warning Nan of Inf, and the validation result becomes nan.
  Excuse me, Does the learning rate too large or other reasons？ thank you.

**Comment by ngoductuanlhp** [comment]
You could try disabling the mixed precision training (by setting `fp16=False` in the config file) and resume training.

**Comment by RongkunYang** [comment]
OK, thank you, I will try it. I debug and find that this is caused by the batchnorm1d in the backbone, the running mean get nan when eval.

**Comment by RobotDogCyberslacking** [comment]
I have also met this issue when training my backbone and load it to finish instance segmentation. 
Disable fp16 can finish training but 24G V-ram  is not enough for batch size 32.

I investigate the pretrained weights you provided, named 'pretrain_stpls3d_val'
unet.u.u.u.u.deconv.0.running_var = tensor(16610601.)
